### ISSUE TITLE
[Bug] Screen rotation icons are not visible properly in dark mode

### ISSUE THREAD

**Issue by imthath-m** [quran/quran-ios#679]
You can see that the portrait and landscape icons still appear in black color even in dark mode and hence not visible clearly (color contrast is bad). Ideally, they should change to white color in dark mode.

![dark](https://github.com/user-attachments/assets/1f9eb1f4-64a4-4709-87eb-32871e086975)
![light](https://github.com/user-attachments/assets/212817fd-b38d-4be1-9b49-9d423d482f34)

**Comment by mohamede1945** [comment]
I've fixed it in https://github.com/quran/quran-ios/commit/3c05f49f3e66991420b936b08130765c85019986
